site/docs/components/_content-template/index.mdx

```diff
@@ -0,0 +1,218 @@
+        - similarTo: This component is similar to the other component (e.g. a
+                     Switch _similarTo_ Checkbox)
+      */
+      relationship: "contains",
+    },
+    {
+      name: "Another Component Name",
+      relationship: "similarTo",
+    },
+    // ...
+  ],
+};
+
+# Component Name
+
+<!--
+  Example screenshot or live component demo here of the default/most common
+  usage.
+
+  For screenshots use Markdown:
+
+  ![Text alternative for screenshot](./path/to/screenshot.jpg)
+
+  For live demos in Storybook use:
+
+  <Canvas>
+    <Story id="..." />
+  </Canvas>
+-->
+
+<!--
+  Introductory text that briefly explains the purpose of the component and
+  anything else of note.
+-->
+
+## When to use Component Name
+
+<!-- Bulleted list of scenarios in which this component can be used  -->
+
+- ...
+- ...
+
+## When not to use Component Name
+
+<!--
+  Bulleted list of scenarios where this component is not appropriate. Where
+  possible, include a link to an alternative component that should be used in
+  it's place
+-->
+
+- ...
+- ...
+
+## Configurations
+
+<!--
+  A visual list of possible configurations of the component.
+
+  The purpose is to convey the main features and/or usage options of the
+  component (e.g. the Data Grid's striped rows might be a "configuration" to
+  showcase here). The configurations listed here should aim to be comprehensive,
+  but not necessarily exhaustive (some complex components may have vast numbers
+  of possible permutations, so listing out each one would be impossible and
+  overwhelming to readers).
+
+  Each confguration should have its own section following the structure outlined
+  below:
+-->
+
+### Configuration title (e.g. Striped rows)
+
+<!--
+  A brief description of what this configuration provides to users and when it
+  should be used.
+-->
+
+<!--
+  A live example of the component configuration, with access to the correspoding
+  source code.
+
+  In Storybook this is done using something like:
+
+  <Canvas>
+    <Story id="..." />
+  </Canvas>
+-->
+
+## Content
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  Add any component-specific content guidance here, use visual examples where
+  appropriate.
+
+  For example, a Button component might include guidance on recommended terms to
+  use for the label text, recommended maximum character counts, etc.
+-->
+
+## Accessibility
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  Describe any considerations the users may need to take into account when using
+  the component.
+-->
+
+## References
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  If there are industry standards for this component, list them in this
+  section. Include links out to useful articles.
+-->
+
+- [Relevant industry standard](https://example-standard.org/)
+- [Useful article](https://some-site.com/article/bla)
+- ...
+
+## Designing
+
+### Theming considerations
```

Comment:
I'm not sure if this needs to be moved to the developing section, what specifically are we going to talk about here? If an example could be provided that would be great. Happy to leave as is for now, but still unclear on what content will live here.

Reply:
As per my earlier comment below, I think we can remove this section altogether.